<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ctpClass</class>
 <widget class="QMainWindow" name="ctpClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1218</width>
    <height>683</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ctp</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>0</y>
      <width>351</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>登  陆</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>行情地址:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="MDEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>交易地址:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="TDEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>BrokerID:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="BIDEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>帐  号:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="USEREdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>密  码:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="PWDEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>28</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_4">
         <property name="text">
          <string>登 陆</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>28</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="Btnxml">
         <property name="text">
          <string>退   出</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>28</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>290</y>
      <width>351</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>下  单</string>
    </property>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>171</width>
       <height>211</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>卖一价：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelAsk">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>最新价：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelLast">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>买一价：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelBid">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>涨停价：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelUp">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>跌停价：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelDown">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="BtnXd">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>240</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>下  单</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget_4" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>191</width>
       <height>251</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>合约代码</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Editdm"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_14">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>下单数量</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="EditLots"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>126</width>
           <height>18</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="QRadioButton" name="radioPJ">
            <property name="text">
             <string>平今仓</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioPZ">
            <property name="text">
             <string>平昨仓</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>171</width>
           <height>22</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_15">
          <item>
           <widget class="QRadioButton" name="radioXJ">
            <property name="text">
             <string>限价</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="EditXj"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>2</x>
           <y>32</y>
           <width>149</width>
           <height>22</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_16">
          <item>
           <widget class="QRadioButton" name="radioSJ">
            <property name="text">
             <string>市价</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>38</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QComboBox" name="comboXd">
            <item>
             <property name="text">
              <string>开  多</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>平  多</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>开  空</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>平  空</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>821</width>
      <height>501</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::West</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>行情</string>
     </attribute>
     <widget class="QTableWidget" name="HQTable">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>811</width>
        <height>471</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>委托</string>
     </attribute>
     <widget class="QTableWidget" name="WTTable">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>811</width>
        <height>471</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>成交</string>
     </attribute>
     <widget class="QTableWidget" name="CJTable">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>1</y>
        <width>801</width>
        <height>471</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>持仓</string>
     </attribute>
     <widget class="QTableWidget" name="ZJTable">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>250</y>
        <width>811</width>
        <height>211</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="CCTable">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>11</y>
        <width>801</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>合约</string>
     </attribute>
     <widget class="QTableWidget" name="HYTable">
      <property name="geometry">
       <rect>
        <x>-5</x>
        <y>1</y>
        <width>811</width>
        <height>471</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>数据</string>
     </attribute>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>3</x>
        <y>0</y>
        <width>801</width>
        <height>501</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_7">
     <attribute name="title">
      <string>日内交易</string>
     </attribute>
     <widget class="QTableWidget" name="DayTable">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>801</width>
        <height>501</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QRadioButton" name="radiorun">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>520</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>全自动交易运行</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radiostop">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>560</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>全自动交易停止</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>521</x>
      <y>598</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>亏损 比例(%):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditpctime">
    <property name="geometry">
     <rect>
      <x>608</x>
      <y>535</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>521</x>
      <y>568</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>保证金比例(%):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditbzj">
    <property name="geometry">
     <rect>
      <x>608</x>
      <y>565</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>521</x>
      <y>537</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>平 仓 时 间:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditloss">
    <property name="geometry">
     <rect>
      <x>608</x>
      <y>595</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditdm">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>506</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>523</x>
      <y>507</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>监 控 代 码:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>510</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>增 加</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>590</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelzt">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>570</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>未开仓</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelyk">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>520</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0.00</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelykbfb">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>550</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0.00</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>550</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1218</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actioncd">
   <property name="text">
    <string>撤单</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="text">
    <string>添加</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ctp.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>ctpClass</receiver>
   <slot>MDLogin()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>983</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel">
     <x>852</x>
     <y>576</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>ctpClass</receiver>
   <slot>OnAddHeyue()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>776</x>
     <y>555</y>
    </hint>
    <hint type="destinationlabel">
     <x>780</x>
     <y>618</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Btnxml</sender>
   <signal>clicked()</signal>
   <receiver>ctpClass</receiver>
   <slot>OnExit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1104</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>1195</x>
     <y>646</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>MDLogin()</slot>
  <slot>OnAddHeyue()</slot>
  <slot>OnExit()</slot>
 </slots>
</ui>
